packages/opencensus-exporter-zpages/src/zpages-frontend/page-handlers/tracez.page-handler.ts

```diff
@@ -91,27 +100,21 @@ const getCanonicalCode = (status: number) => {
  * This change should be made in the core module.
  * @param inputSpan The span to serialize.
  */
-function serializeSpan(inputSpan: Span|RootSpan): Partial<Span> {
-  const span: Partial<Span> = {
+function serializeSpan(inputSpan: Span): SerializedSpan {
+  const span: SerializedSpan = {
     startTime: inputSpan.startTime,
     duration: inputSpan.duration,
     traceId: inputSpan.traceId,
     id: inputSpan.id,
-    parentSpanId: inputSpan.id
+    parentSpanId: inputSpan.id,
```

Comment:
Can this be `inputSpan.parentSpanId`, I know you didn't start the fire but looks suspicious.

Reply:
Good catch.  Done.